src/wp-includes/load.php

```diff
@@ -753,7 +753,7 @@ function wp_start_object_cache() {
 			)
 		);
 
-		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins' ) );
+		wp_cache_add_non_persistent_groups( array( 'counts', 'plugins', 'theme_json' ) );
```

Comment:
If https://github.com/WordPress/wordpress-develop/pull/3712 lands first, this would have already been ported over there (same for the other places where the group was added).

Reply:
Same question here https://github.com/WordPress/wordpress-develop/pull/3556#discussion_r1052602544.

Reply:
responded at https://github.com/WordPress/wordpress-develop/pull/3556#issuecomment-1361689398

Reply:
See above, IMO this is good to keep.